plugins.json

```diff
@@ -83,6 +83,18 @@
         "remote_install": true,
         "compatible_version": ">=1.0.32"
     },
+    "bigboard": {
+        "title": "Bigboard",
+        "version": "1.0.2",
+        "author": "Stinnux",
+        "license": "MIT",
+        "description": "A Kanboard that can display multiple projects.",
+        "homepage": "https://github.com/stinnux/kanboard-bigboard",
+        "readme": "https://raw.githubusercontent.com/stinnux/kanboard-bigboard/master/README.md",
+        "download": "https://github.com/stinnux/kanboard-bigboard/releases/download/1.0.2/Bigboard-1.0.2.zip",
+        "remote_install": true,
+        "compatible_version": ">=1.1.22"
```

Comment:
There is a typo here, version 1.1.22 doesn't exists.

Reply:
You are right, tyipo fixed.